automate the zenodo bibtex (#880)
pynucastro/pynucastro: CITATION.md

Zenodo DOI.  You can also cite the JOSS paper.

> ## Astrophysical Journal ("2.0 paper":)

Reviewer: ```suggestion
## Astrophysical Journal ("2.0 paper")
```

https://iopscience.iop.org/article/10.3847/1538-4357/acbaff

> ```

Reviewer: This makes the syntax highlighting match the included file
```suggestion
```bibtex
```

```

## JOSS

https://joss.theoj.org/papers/10.21105/joss.00588

> ```

Reviewer: ```suggestion
```bibtex
```

}
```

> ## Zenodo

Reviewer: We should also keep the Zenodo DOI link, as mentioned at the top.
```suggestion
## Zenodo

https://doi.org/10.5281/zenodo.1202434
```

}
```

## Zenodo

https://doi.org/10.5281/zenodo.1202434

> The bibtex for the latest version is always here:
> https://zenodo.org/records/13899727/export/bibtex

Reviewer: Since `https://zenodo.org/records/1202434/export/bibtex` doesn't work, maybe something like this is better?
```suggestion
The bibtex for the latest version is included below (updated automatically):
```

Author: oh wait...

that's in `CHANGES.md`, so if someone views that independent of the docs, then it won't appear

Author: yeah, I'll do that
